dataactcore/models/domainModels.py

```diff
@@ -25,4 +25,14 @@ class ObjectClass(Base):
     __tablename__ = "object_class"
     object_class_id = Column(Integer, primary_key=True)
     object_class_code = Column(Text,nullable=False,index=True,unique=True)
-    object_class_name = Column(Text)
\ No newline at end of file
+    object_class_name = Column(Text)
+
+class ProgramActivity(Base):
```

Comment:
Is there a combination of fields in this table that would qualify for a unique index?